primitives/core/src/assertion/mod.rs

```diff
@@ -17,14 +17,33 @@
 // This file includes the predefined rulesets and the corresponding parameters
 // when requesting VCs.
 
-use crate::{
-	all_web3networks, AccountId, AchainableParams, BnbDigitDomainType, BoundedWeb3Network,
-	EVMTokenType, GenericDiscordRoleType, OneBlockCourseType, ParameterString, PlatformUserType,
-	VIP3MembershipCardLevel, Web3Network, Web3NftType, Web3TokenType,
-};
+pub mod achainable;
+pub mod bnb_domain;
+pub mod contest;
+pub mod evm_amount_holding;
+pub mod generic_discord_role;
+pub mod network;
+pub mod oneblock;
+pub mod platform_user;
+pub mod soraquiz;
+pub mod vip3;
+pub mod web3_nft;
+pub mod web3_token;
+
+use crate::{AccountId, ParameterString};
+use achainable::AchainableParams;
+use bnb_domain::BnbDigitDomainType;
+use evm_amount_holding::EVMTokenType;
+use generic_discord_role::GenericDiscordRoleType;
+use network::{all_web3networks, BoundedWeb3Network, Web3Network};
+use oneblock::OneBlockCourseType;
 use parity_scale_codec::{Decode, Encode, MaxEncodedLen};
+use platform_user::PlatformUserType;
 use scale_info::TypeInfo;
 use sp_std::{vec, vec::Vec};
+use vip3::VIP3MembershipCardLevel;
+use web3_nft::Web3NftType;
+use web3_token::Web3TokenType;
```

Comment:
Is it possible to separate those imports at least under:
- crate level (`use crate::...`)
- local (sub) mod (e.g. `network::...`)
- extern crates (e.g. `sp_std`, `parity_scale_codec` etc